Document the k6/experimental/webcrypto module (#1119)
grafana/k6-docs: src/data/markdown/docs/02 javascript api/07 k6-experimental/06 webcrypto/01 Crypto/01 getRandomValues.md

```
getRandomValues(typedArray)
```

## Parameters

| Name         | Type         | Description                                                                                                                                                                                                      |
| :----------- | :----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `typedArray` | `TypedArray` | An integer-based `TypedArray` to fill with random values. Accepted `TypedArray` specific types are: `Int8Array`, `Uint8Array`, `Uint8ClampedArray`, `Int16Array`, `Uint16Array`, `Int32Array`, or `Uint32Array`. |

## Return Value

The same array passed as the `typedArray` parameter with its contents replaced with the newly generated random numbers. the `typedArray` parameter is modified in place, and no copy is made.

## Throws

| Type                 | Description                                                          |
| :------------------- | :------------------------------------------------------------------- |

> | `QuotaExceededError` | if the `typedArray` is too large and its `byteLength` exceeds 65536. |

Reviewer: ```suggestion
| `QuotaExceededError` | Thrown when `typedArray` is too large and its `byteLength` exceeds 65536. |
```
